recipes-ni/ni-utils/ni-utils_1.0.bb

```diff
@@ -11,7 +11,10 @@ SRC_URI = "file://status_led \
 "
 
 FILES_${PN} += "\
-	/usr/lib/nisetbootmode.functions \
+	${bindir}/status_led \
```

Comment:
Shouldn't these LED changes be in the previous commit?

Reply:
These changes aren't specifically for LED, I've replaced paths for all files in the package and it seemed better to do them together rather than breaking them up.

Reply:
Moved them to previous commit as per offline conversation.